src/MockBot.Api/Controllers/DmrController.cs

```diff
@@ -36,19 +38,28 @@ public async Task<ActionResult> PostDmrMessageAsync([FromHeader] HeadersInput he
                 }
 
                 var decodedPayload = _encoder.DecodeBase64(encodedPayload);
+                var payload = JsonSerializer.Deserialize<DmrRequestPayload>(decodedPayload);
 
-                // Just log telemetry for the DMR Callback at the moment...
+                // Add the message to the chat
+                var chat = _chatService.FindByMessageId(new Guid(headers?.XMessageIdRef));
+                if (chat == null)
+                {
+                    // No matching message, create a new chat
```

Comment:
Hmmm - we just spoke about this.  This might be completely legit...  If a bot forwards a message to another bot - there won't be a chat.  This makes perfect sense, actually...  Ignore my musings...

Reply:
@joonasroosalung - you probably have a bit more familiarity with the bot what do you think should happen?

Reply:
Have added a new logger extension to log that a message with no chat has arrived and go ahead and create a new chat for it.